auto_py_to_exe/web/js/i18n.js

```diff
@@ -507,6 +542,7 @@ const translationMap = {
                 fr: 'Injectez du texte brut dans la commande générée.',
                 it: 'Inserisci testo forzatamente alla fine del comando generato.',
                 id: 'Masukkan teks mentah ke dalam command yang dibuat',
+                cz: 'Vložit text do vygenerovaného příkazu.'

```

Comment:
Can we get a comma at the end of this line please

Reply:
Fixed!